src/style/fims/_query.scss

```diff
@@ -87,59 +87,49 @@ $query-form-toggle-width: 40px;
   > form {
     height: 100%;
   }
-  section {
-    margin-bottom: $layout-gutter-width / 2;
 
-    .md-form-group {
-      margin-bottom: 0;
-    }
-
-    > a:focus,
-    > a:hover {
-      text-decoration: none;
-    }
+  h3 {
+    margin: 0px;
   }
 
-  .md-input-container-dense {
-    md-input-container {
-      margin: 0;
-      margin-top: 5px;
+  md-checkbox {
+    margin-bottom: 0px;
+  }
 
-      .md-errors-spacer {
-        min-height: 0;
-      }
+  .term {
+    position: relative;
+    button {
+      position: absolute;
     }
   }
 
-  .sequences {
-    @extend .md-input-container-dense;
+  .country,
+  .term,
+  .phylum,
+  .year,
+  .groups,
+  .projects {
+    padding-top: 0px;
+    padding-bottom: 0px;
```

Comment:
so classes are meant to be reused. It would be better to create a single class and use that in the html, instead of all these classes specifying the same css rules

Reply:
It looks like this CSS wasn't being used anyways. I went ahead and removed all of the classes.